docs/services/slam/cartographer/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Cartographer Integrated Library"
+linkTitle: "cartographer"
+weight: 70
+type: "docs"
+description: "Configure a SLAM service with the Cartographer library."
+tags: ["slam", "services"]
```

Comment:
```suggestion
tags: ["slam", "services"]
empty_node: true
```

this will make the page just a toggle on the side bar.

I'd suggest moving the paragraph onto each page with an include or onto the SLAM page. That's then fewer clicks for the user and the information stays together.

However, that may not be the correct thing to do. Between the live mode and offline mode there's now a lot of duplication. Would it make more sense to put the differing data between live mode and offline mode into tabs and otherwise keep everything on one cartographer page?

Introducing Life and offline is something I think we still need to do better, right?

Reply:
@npentrel Yeah, I was thinking about that as well and that seems like a good idea. I'll see if I can make it work without looking too messy (+ moving the data_dir and mapping info into duplicates on the library pages)

Reply:
@npentrel Is this change still necessary? Sorry I am not sure as to original suggestion

Reply:
no - this was an either or :) - looking good I think